### DISCUSSION TITLE
Build Platformio

### DISCUSSION THREAD

**Discussion by romi06** [pixelit-project/PixelIt#237] [General]
Could not find remote branch esp32s2 to clone

**Comment by foorschtbar** [comment]
There is no for esp32s2. feel free to create one. PRs are welcome :)

**Comment by romi06** [comment]
Translate Google
Yes it is not for esp32s2.
This is the message I get when compiling for "wemos_d1_mini32"
platformio.ini
wemos_d1_mini32
........
WiFiManager = https://github.com/tzapu/WiFiManager.git#esp32s2

How do I change to 
"WiFiManager = https://github.com/tzapu/WiFiManager.git"

This gets 
".pio/libdeps/wemos_d1_mini32/WiFiManager/WiFiManager.cpp: In member function 'bool WiFiManager::WiFiSetCountry()':
.pio/libdeps/wemos_d1_mini32/WiFiManager/WiFiManager.cpp:3531:11: error: 'esp_wifi_set_country_code' was not declared in this scope
      err = esp_wifi_set_country_code(_wificountry.c_str(), true);"
